src/core/grade_notifier.py

```diff
@@ -176,18 +176,31 @@ def welcome_message():
         .newline() \
         .add("You're all set up. You should recieve a message soon with your current grades.") \
         .newline() \
-        .add("After that first message, the notifier will message you whenever a grade changes (or is added)!")
-        .newline() \
+        .add("After that first message, the notifier will message you whenever a grade changes (or is added)!") \
+        .newline()
     return new_message.sign().message()
 
 
-def create_instance(retry = True):
+MAX_SIGN_IN_ATTEMPTS = 5
+
+
+def sign_in(attempt=0):
```

Comment:
I rather this move from 5 ~> 0 then from 0 ~> 5. That way we can change in the function call how many attempts

Reply:
The default can be MAX_SIGN_IN_ATTEMPTS